src/index.html

```diff
@@ -7,6 +7,68 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--mon calendar--margin--auto">
+      <div class="calendar__day calendar__day--1"></div>
```

Comment:
u should use only one class for days, without numerating, use loops & nth-child instead

Reply:
Done